### PR TITLE
URE CrowdStrike Identity Protect 1.0.0-beta - Initial release

### PR DESCRIPTION
**Pull request by cwadhwani-crest**

### CODE REVIEW

File: crowdstrike_identity_protect/lib/falconpy/_endpoint/_falconx_sandbox.py [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+        "type": "string",
+        "description": "The file SHA256.",
+        "name": "ids",
+        "in": "query",
+        "required": True
+      },
+      {
+        "type": "boolean",
+        "default": False,
+        "description": "Flag whether the sample should be zipped and password protected with pass='infected'",
```
Comment (ns-snekkar) [inline crowdstrike_identity_protect/lib/falconpy/_endpoint/_falconx_sandbox.py:273]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: crowdstrike_identity_protect/lib/falconpy/_endpoint/_sample_uploads.py [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+        "type": "string",
+        "description": "The file SHA256.",
+        "name": "ids",
+        "in": "query",
+        "required": True
+      },
+      {
+        "type": "boolean",
+        "default": False,
+        "description": "Flag whether the sample should be zipped and password protected with pass='infected'",
```
Comment (ns-snekkar) [inline crowdstrike_identity_protect/lib/falconpy/_endpoint/_sample_uploads.py:294]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

Comment (cwadhwani-crest) [reply crowdstrike_identity_protect/lib/falconpy/_endpoint/_sample_uploads.py:294]: This is a sample upload file of the falconpy SDK library.

File: crowdstrike_identity_protect/lib/falconpy/_endpoint/_falconx_sandbox.py (hunk 1 above)
Comment (cwadhwani-crest) [reply crowdstrike_identity_protect/lib/falconpy/_endpoint/_falconx_sandbox.py:273]: This is a sample upload file of the falconpy SDK library.

**Pull request merged**